test_scenarios/send_data.py

```diff
@@ -8,6 +8,10 @@
 from os import path
 import uuid
 import yaml
+from botocore.config import Config
+
+from global_helpers import panther_base_helpers
```

Comment:
This is not a detection, it's a script to load dummy data for scenarios and the like. I don't think we need to make any changes here, and I'm 99% sure that `from global_helpers import panther_base_helpers` won't work.